src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.springboot.datajpa.controller;
+
+import mate.academy.springboot.datajpa.dto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import mate.academy.springboot.datajpa.service.mapper.CategoryMapper;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/categories")
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final CategoryMapper mapper;
+
+    public CategoryController(CategoryService categoryService, CategoryMapper mapper) {
+        this.categoryService = categoryService;
+        this.mapper = mapper;
+    }
+
+    @PostMapping
+    public CategoryResponseDto save(@RequestBody CategoryRequestDto requestDto) {
+        Category category = categoryService.save(mapper.fromDto(requestDto));
+        return mapper.toDto(category);
+    }
+
+    @GetMapping("/{id}")
+    public CategoryResponseDto getById(@PathVariable Long id) {
+        Category category = categoryService.get(id);
+        return mapper.toDto(category);
+    }
+
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        categoryService.delete(id);
+    }
+
+    @PutMapping
```

Comment:
I think that we should also use PathVariable, because how will Spring know which category should be updated if we don't give it an id?